xml/admin_ceph_upgrade.xml

```diff
@@ -1397,6 +1397,147 @@ data:
+   example faster, media.
+  </para>
+  <para>
+   In &productname; 5, both &filestore; and &bluestore; are supported and it is possible
+   for &filestore; and &bluestore; OSDs to co-exist in a single cluster. During
+   the &productname; upgrade procedure, &filestore; OSDs are not
+   automatically converted to &bluestore;. Be aware that the
+   &bluestore;-specific features will not be available on OSDs that have not
+   been migrated to &bluestore;.
+  </para>
+  <para>
+   Before converting to &bluestore;, the OSDs need to be running &productname; 5.
+   The conversion is a slow process as all data gets
+   re-written twice. Though the migration process can take a long time to
+   complete, there is no cluster outage and all clients can continue accessing
+   the cluster during this period. However, do expect lower performance for
+   the duration of the migration. This is caused by rebalancing and
+   backfilling of cluster data.
+  </para>
+  <para>
+   Use the following procedure to migrate &filestore; OSDs to &bluestore;:
+  </para>
+  <tip>
+   <title>Turn Off Safety Measures</title>
+   <para>
+    &salt; commands needed for running the migration are blocked by safety
+    measures. In order to turn these precautions off, run the following
+    command:
+   </para>
+ <screen>
+ &prompt.smaster;salt-run disengage.safety
+ </screen>
+   <para>Rebuild the nodes before continuing:</para>
+ <screen>
+ &prompt.smaster; salt-run rebuild.node <replaceable>TARGET</replaceable>
+ </screen>
+   <para>You can also choose to rebuild each node individually. For example:</para>
+ <screen>
+&prompt.smaster; salt-run rebuild.node data1.ceph
+ </screen>
+   <para>The <literal>rebuild.node</literal> always removes and recreates
+     all OSDs on the node.</para>
+     <important>
+       <para>If one OSD fails to convert, re-running the rebuild destroys the
+         already converted &bluestore; OSDs. If this happens, run:</para>
+         <screen>
+           &prompt.smaster; salt-run disks.deploy <replaceable>TARGET</replaceable>
+         </screen>
+       </important>
+ </tip>
+  <procedure>
+   <step>
+    <para>
+     Migrate hardware profiles:
+    </para>
+ <screen>&prompt.smaster;salt-run state.orch ceph.migrate.policy</screen>
+    <para>
+     This runner migrates any hardware profiles currently in use by the
+     <filename>policy.cfg</filename> file. It processes
+     <filename>policy.cfg</filename>, finds any hardware profile using the
+     original data structure, and converts it to the new data structure. The
+     result is a new hardware profile named
+     'migrated-<replaceable>ORIGINAL_NAME</replaceable>'.
+     <filename>policy.cfg</filename> is updated as well.
+    </para>
+    <para>
+     If the original configuration had separate journals, the &bluestore;
+     configuration will use the same device for the 'wal' and 'db' for that
+     OSD.
+    </para>
```

Comment:
I just double-checked.  While the state `ceph.migrate.policy` is there, the referenced module call no longer exists in SES6.  

This paragraph can be removed.